vignettes/grouping.Rmd

```diff
@@ -0,0 +1,172 @@
+---
+title: "Grouping behavior in yardstick"
+author: "Simon Couch"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Grouping behavior in yardstick}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+The 1.3.0 release of yardstick introduced an implementation for _group-wise metrics_. The use case motivating the implementation of this functionality is _fairness metrics_, though group-wise metrics have applications beyond that domain. Fairness metrics quantify the degree of disparity in a metric value across groups. To learn more about carrying out fairness-oriented analyses with tidymodels, see the blog post on the tidymodels website. This vignette will instead focus on group-wise metrics generally, clarifying the meaning of "group-wise" and demonstrating functionality with an example dataset.
+
+<!-- TODO: link to forthcoming tidymodels blog post -->
+
+```{r pkgs, message = FALSE}
+library(yardstick)
+library(dplyr)
+
+data("hpc_cv")
+```
+
+# Group-awareness
+
+Even before the implementation of group-wise metrics, _all_ yardstick metrics had been _group-aware_. By group-aware, we mean that when passed grouped data, a metric will return metric values calculated for each group.
```

Comment:
I understand what it meant with "when passed grouped data", but I think it takes a couple of reads to understand.

What do you think about the following rephrase? 

```suggestion
Even before the implementation of group-wise metrics, _all_ yardstick metrics had been _group-aware_. When grouped data is passed to a group-aware metric, it willwill return metric values calculated for each group.
```